src/git-lfs/install.sh

```diff
@@ -15,6 +15,7 @@ GIT_LFS_ARCHIVE_GPG_KEY_URI="https://packagecloud.io/github/git-lfs/gpgkey"
 GIT_LFS_ARCHIVE_ARCHITECTURES="amd64 arm64"
 GIT_LFS_ARCHIVE_VERSION_CODENAMES="stretch buster bullseye bionic focal jammy"
 GIT_LFS_CHECKSUM_GPG_KEYS="0x88ace9b29196305ba9947552f1ba225c0223b187 0x86cd3297749375bcf8206715f54fe648088335a9 0xaa3b3450295830d2de6db90caba67be5a5795889"
+KEYSERVER_PROXY="${HTTP_PROXY:-""}"
```

Comment:
@rbambrough-intel Thank you so much for taking the time to contribute this PR.

Instead of these changes, how about we making similar changes https://github.com/devcontainers/features/pull/1119 ?

Reply:
It seems the link is not pointing to the right PR and I don't understand your statement as it seems to be the common way to define a default variable value in this project (e.g.: https://github.com/devcontainers/features/blob/0be79410981d50c2b5fa35e97f4af8a1a4f4727d/src/ruby/install.sh#L18, https://github.com/devcontainers/features/blob/302fecab063c63a4f3b6de1bbba408f4b896f7cd/src/java/install.sh#L29, https://github.com/devcontainers/features/blob/302fecab063c63a4f3b6de1bbba408f4b896f7cd/src/powershell/install.sh#L16). However I think this following will be enough:

```suggestion
KEYSERVER_PROXY="${HTTP_PROXY}"
 ```

 But maybe I missed something.

Reply:
Apologies, I pasted the incorrect link.

I was recommending https://github.com/devcontainers/features/pull/1124 which tests the servers before using them to download the keys.

> KEYSERVER_PROXY="${HTTP_PROXY:-""}"

This simply assigns the `KEYSERVER_PROXY` to "" if `HTTP_PROXY` is not defined.

Reply:
@Jarsop If we are fine with removing the httpProxy setting from some of the features then it can be simplified into:

```bash
KEYSERVER_PROXY="$HTTP_PROXY"
```

Reply:
@samruddhikhandale It does do curl checking of the key servers, just not the same as the one you linked (map instead of multiple function calls). I wasn't sure touching the function one so I adjusted all the others that looked closee